src/Tizen.NUI/src/public/Layer.cs

```diff
@@ -74,8 +74,6 @@ public enum LayerBehavior
             /// UI control rendering mode.
             /// </summary>
             /// <since_tizen> 3 </since_tizen>
-            [Obsolete("Please do not use! This will be deprecated! Please use LayerUI property instead!")]
-            [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
But I really want this to be deprecated :)

We should be using LayerUI now and I think we should try and get them to stop using Layer2D .

Reply:
What is the process for an ACR in C#? Can't we just make the change in Samsung/TizenFX or are you waiting to do the change at the end?

If you're doing it at the end isn't it good that the dalihub/TizenFX version is different so that it reminds us of what APIs we need to change? I'm not sure if that becomes a maintenance burden though.

Maybe we can discuss the workflow of this on Thursday.

Reply:
Yes. You are right. It would be good to maintain some distinguishable tag to keep remind of ACR. However, as you can see below, the TizenAPI-Bot attaches the "ACR Required" Red warning label when the API Compatibility violation exists in the patch uploaded. Also because of the TCT process, we can not change the codes before ACR (We got several warnings from SQE team in the past).

So I usually do like this and also will do later for deprecated ones.
  - Adding : 
    1. Attach [EditorBrowsable(EditorBrowsableState.Never)] attribute to be treated as HiddenAPI(InhouseAPI) then proceed ACR process. 
    2. After ACR accepted, remove [EditorBrowsable(EditorBrowsableState.Never)] attribute. If TizenAPI-Bot gives warning, comment ACR link on the patch.
    3. Make TCT. (example : https://review.tizen.org/gerrit/#/c/test/tct/csharp/api/+/210802/ )
    4. Add or Enrich comments. Upload guide in tizen-doc if required. (example : https://github.com/Samsung/tizen-docs/pull/805 )

  - Removing : 
    1. Proceed ACR process first to be accepted then attach  [Obsolete("xxx ... deprecated version : API6")] attribute.
    2. Even if TizenAPI-Bot gives API break warning, it can be ignored by showing ACR link which is accepted. 
    3. Maintain the API for 2 versions of tizen release. (example : API6(decide deprecated), API7(maintained), API8(maintained), API9(actually removed) )
    4. Remove TCT item.

If you find any problem on the steps above. please let me know.
Thank you.

Reply:
Hi Ssong,

Thanks for your answer. And yes I understand why you are doing it but those rules only apply to Samsung/TizenFX right.

On dalihub/TizenFX we can keep it different so that we remember that we do have to change it in the future :)

I don't mind either way, I was just suggesting a slightly different approach.

It's entirely up to you.

Many Thanks,
Adeel

Reply:
Thank you for suggestion. I want to maintain dalihub/TizenFX/devel/master and samsung/TizenFX/master are same by regular integration work (just same as dali native). So I prefer to add [Obsolete(...)] after ACR done. Thank you for understanding.